File: Copy_Small_String/Copy.py
"""
    Author : Pakshal Shashikant Jain
    Date : 25/08/2023
    Program : Write a program which accept string from user and copy small
              characters of that string into another string.

              Input : "Marvellous multi OS"
              Output : "arvellous multi"
"""

########################################################################################################################################

#Class Declaration
class CopyString :
    #Characteristics of Class
    arr = [];
    brr = [];
    Str = " ";
    
    #Constructor of Class
    def __init__(self) :
        self.arr = [];
        self.brr = [];
        self.Str = " ";

    #Behaviour of CLass To Take Input from user
    def Accept(self) :
        #Taking Input from user
        print("Enter one String");
        self.Str = input();

        self.arr = list(self.Str);

    #Behaviour of Class To Copy Small String from Entered String into Another String
    def StrCpySmall(self) :
        i = 0;
        
        #Logic To Copy String
        while(i != len(self.arr)) :
            if(self.arr[i] == ' ') :
                self.brr.append(self.arr[i]);

            if((self.arr[i] >= 'a')and(self.arr[i] <= 'z')) :
                self.brr.append(self.arr[i]);
            
            i = i + 1;

#Main Function
def main() :
    print("Jay Ganesh....");

    #Creating object of CopyString Class 
    cobj = CopyString();

    #Call To Behaviours of Class
    cobj.Accept();
    cobj.StrCpySmall();

    #Printing Small Letters Copied
    print("Small Letters Copied From String is : %s"%"".join(cobj.brr));

#########################################################################################################################

#Call Back Method To Call Main Function
if __name__ == "__main__" :
    #Call To Main Function
    main();

File: Copy_Small_String/Copy.java
/*
    Author : Pakshal Shashikant Jain
    Date : 25/08/2023
    Program : Write a program which accept string from user and copy small
              characters of that string into another string.

              Input : "Marvellous multi OS"
              Output : "arvellous multi"
*/

////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

//Required Libraries
import java.lang.*;
import java.util.*;

//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

//Class Declaration
class CopyString 
{
    //Characteristics of Class
    char arr[];
    char brr[];
    String str;

    //Constructor of Class
    CopyString()
    {
        this.arr = null;
        this.brr = null;
        this.str = " ";
    }

    //Behaviour of Class To Take Input from user
    void Accept()
    {
        //Creating object of Scanner Class To Take Input from user
        Scanner sobj = new Scanner(System.in);

        //Taking Input from user
        System.out.println("Enter One String");
        this.str = sobj.nextLine();

        //Allocating Memory to array
        this.arr = new char[this.str.length()];
        this.brr = new char[this.str.length()];

        //Creating Character Array of Enterd String
        this.arr = this.str.toCharArray();
    }

    //Behaviour of Class To Copy Small Characters of Entered String 
    void StrCpySmall()
    {
        int i = 0;

        if((this.arr == null)&&(this.brr == null))
        {
            return;
        }

        //Logic To Copy String
        while(i != this.arr.length)
        {
            if(this.arr[i] == ' ')
            {
                this.brr[i] = ' ';
            }
            
            if((this.arr[i] >= 'a')&&(this.arr[i] <= 'z'))
            {
                this.brr[i] = this.arr[i];
            }

            i++;
        }
    }
}

//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

//Main Class
class Copy
{
    //Main Function
    public static void main(String args[])
    {
        System.out.println("Jay Ganesh....");

        //Creating object of CopyString Class
        CopyString cobj = new CopyString();

        //Call To Behaviour of Class
        cobj.Accept();
        cobj.StrCpySmall();

        //Printing Copied Small Characters From Entered String
        System.out.print("Small Letters Copied From Entered String into Another String : ");
        System.out.print(cobj.brr);
    }
}